File: c_practice_file/leap_year.c
#include<stdio.h>
void main()
{
 int x,y ;
 printf(" enter the year ");
 scanf("%d",&x);
 y=x%4;
 printf(" reaminder %d\n",y);
 if(x%4==0 && x%400==0)
 {
 printf(" leap year\n");
 }
 else
 printf("not\n");
 }

File: python_lab/class_assignment/leap_year.py
'''
Wap to calculate Leap year or not?
'''

print("\nWap to calculate Leap year or not?\n")
y=int(input("Enter year -\t"))
if((y % 400 == 0) or (y % 100 != 0) and (y % 4 == 0)):
    print("Year is leap Year",y)
else:
    print("Year isn't leap Year",y) 